clients/imodels-client-authoring/README.md

```diff
@@ -0,0 +1,37 @@
+# @itwin/imodels-client-authoring
+
+Copyright © Bentley Systems, Incorporated. All rights reserved. See [LICENSE.md](../../LICENSE.md) for license terms and full copyright notice.
+
+## About this package
+
+This package contains an API client that exposes a subset of [iModels API](https://developer.bentley.com/apis/imodels/) operations - it extends the client from the [`@itwin/imodels-client-management`](../imodels-client-management/README.md) package and adds more operations that enable applications to author iModels - acquire Birefcases, manage Locks, etc. This library also adds the ability to perform iModel operations that include file transfer - create or download Changesets, create iModel from Baseline file.
```

Comment:
We should mention pushing changesets as authoring capabilities.

Reply:
It is included:
https://github.com/iTwin/imodels-clients/blob/8d1ba8536b41d0b6bb1aac2118b9ac456c6a0324/clients/imodels-client-authoring/README.md?plain=1#L7
![image](https://user-images.githubusercontent.com/70565417/145415758-ae2dee16-6ea3-4a0d-b477-f8e0a3eeda46.png)

Reply:
Missed that